src/Microsoft.AspNet.Mvc.Core/ModelBinding/MutableObjectModelBinder.cs

```diff
@@ -359,7 +365,7 @@ protected virtual object GetModel([NotNull] ModelBindingContext bindingContext)
                 var modelMetadataPredicate = context.ModelMetadata.PropertyBindingPredicateProvider?.PropertyFilter;
 
                 return
-                    context.PropertyFilter(context, propertyName) &&
+                    (context.PropertyFilter == null || context.PropertyFilter(context, propertyName)) &&
                     (modelMetadataPredicate == null || modelMetadataPredicate(context, propertyName));
```

Comment:
If a `null` `PropertyFilter` is :ok:, why initialize the property to `_defaultPropertyFilter`? Suggest removing `_defaultPropertyFilter`.